src/components/interaction-designer/Block.vue

```diff
@@ -390,27 +399,44 @@ export class Block extends mixins(Lang) {
   @builderNamespace.Action setConnectionCreateTargetBlockToNullFrom!: BlockAction
   @builderNamespace.Action applyConnectionCreate!: () => void
 
+  UpdateShouldShowBlockToolBar(): void {
+    //do not show the block toolbar when waiting for connection
```

Comment:
This is what is designed in the "block states dsn"

Reply:
@object-required to use camelCase here